app/views/admins/registrations/new.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="loginPage">
```

Comment:
If it's not possible to remote the `admins/sign_up` route altogether, let's get rid of the HTML. We don't want any user to "find" the URL and make admin accounts :O